frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/models.ts

```diff
@@ -0,0 +1,1362 @@
+import { K8sModel } from '../../api/common-types';
+
+export const PrometheusModel: K8sModel = {
```

Comment:
Models are for sure problematic... not sure what our solution is for models for things. I guess we are over extending if we are moving things that are Prometheus into the SDK... because Prometheus doesn't really belong in either SDK I don't feel.

Reply:
Yeah, I'm not sure we need this file... can you help me understand why we need these models?

Reply:
Fixed WRT our discussion this morning - leaving models in the console and importing into the SDK.